theme-default/gulpfile.js

```diff
@@ -3,18 +3,16 @@ const gulp = require('gulp'),
 	del = require('del'),
 	autoprefixer = require('autoprefixer'),
 	browserSync = require('browser-sync').create(),
-	runSequence = require('run-sequence'),
 	rmHover = require('postcss-hover'),
-	inlinesvg = require('postcss-inline-svg');
-
-/**
- * Load gulp prefixed plugins from our package.json
- * Plugins can be loaded with: load.name(), rather than just name().
- * More information: https://w+ww.npmjs.com/package/gulp-load-plugins
- */
-const load = require('gulp-load-plugins')();
-
-const project = 'dna-recipe', // Your project name
+	inlinesvg = require('postcss-inline-svg'),
```

Comment:
I prefer keeping project config seperate from the module imports. They may both be `const` but they are for very different purposes.